weave/logs.py

```diff
@@ -237,28 +242,28 @@ def configure_logger() -> None:
             # we don't want the logs to show up in the notebook.
             enable_stream_logging(
                 logger,
-                pid_logfile_settings=LogSettings(LogFormat.PRETTY, "INFO"),
+                pid_logfile_settings=LogSettings(LogFormat.PRETTY, logging.INFO),
             )
     else:
         if dd_env == "ci":
             # CI expects logs in the pid logfile
             enable_stream_logging(
                 logger,
-                wsgi_stream_settings=LogSettings(LogFormat.PRETTY, "INFO"),
-                pid_logfile_settings=LogSettings(LogFormat.PRETTY, "INFO"),
+                wsgi_stream_settings=LogSettings(LogFormat.PRETTY, logging.INFO),
+                pid_logfile_settings=LogSettings(LogFormat.PRETTY, logging.INFO),
             )
         elif environment.wandb_production():
             # Only log in the datadog format to the wsgi stream
             enable_stream_logging(
                 logger,
-                wsgi_stream_settings=LogSettings(LogFormat.DATADOG, "DEBUG"),
+                wsgi_stream_settings=LogSettings(LogFormat.DATADOG, level=None),
             )
         else:
             # Otherwise this is dev mode with datadog logging turned on.
             # Log to standard out and a known filename that the datadog
             # agent can watch.
             enable_stream_logging(
                 logger,
-                wsgi_stream_settings=LogSettings(LogFormat.PRETTY, "DEBUG"),
-                server_logfile_settings=LogSettings(LogFormat.DATADOG, "DEBUG"),
+                wsgi_stream_settings=LogSettings(LogFormat.PRETTY, level=None),
+                server_logfile_settings=LogSettings(LogFormat.DATADOG, level=None),
```

Comment:
Since `DEBUG` is the most permissive level, it's essentially the same as the handler just following the logger's level.